python-connectors/googlesheets-sheet/connector.py

```diff
@@ -3,20 +3,26 @@
 from collections import OrderedDict
 from gspread.utils import rowcol_to_a1
 from slugify import slugify
-from googlesheets import get_spreadsheet
+from googlesheets import GoogleSheetsSession
+from safe_logger import SafeLogger
+
+
+logger = SafeLogger("googlesheets plugin", ["credentials"])
+
 
 class MyConnector(Connector):
```

Comment:
Discussed with @alexbourret  - I think the get_records_count()  code will no longer be relevant as "canCountRecords" is being set to false, but:
- I'm not sure which PR/commit sets "canCountRecords" to false? Could you add a link to that here?
- we should probably still comment about this here or remove this code as it could be confusing (doesn't need to be in this PR but we could link to it here).

Reply:
@liamlynch-data canCountRecords was set to false in the Release/1.2.0 breanch, in [this commit](https://github.com/dataiku/dss-plugin-googlesheets/commit/9a1aa2b3f168c460b0b391308a99e9aaa2a67524)
I'll add a comment in the get_records_count method in Release/1.2.0